pipeline/types/notifications.py

```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+
+from pipeline.types.chat import ChatData
+from pipeline.types.match import Match
+
```

Comment:
Consider adding a type for the output notification. What information do we need to give the system that is sending the notification?

- User to send to
- Email of user to send to (this is not on the `User` dataclass currently, but we will add it shortly)
- Content of notification
- Etc...

Reply:
This shouldn't be necessary anymore, right? Because the User datatype has been refactored to include email and the like. We'd just return the User object and, if that information is needed, we pull it directly from it.